### PR TITLE
LPS-137177

### PR DESCRIPTION
**Pull request by mlan0**

### CODE REVIEW

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/form_instance_records_search_container.jsp [hunk 1]
```diff
@@ -125,8 +125,9 @@ PortletURL portletURL = ddmFormViewFormInstanceRecordsDisplayContext.getPortletU
 						</c:when>
 						<c:when test="<%= StringUtil.equals(ddmFormField.getType(), DDMFormFieldTypeConstants.CHECKBOX) %>">
 							<c:choose>
-								<c:when test='<%= StringUtil.equals(ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(ddmFormFieldsMap.get(ddmFormField.getName()), StringPool.BLANK, ddmFormFieldValuesMap.get(ddmFormField.getName())), "Yes") %>'>
+								<c:when test='<%= StringUtil.equals(ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(ddmFormFieldsMap.get(ddmFormField.getName()), StringPool.BLANK, ddmFormFieldValuesMap.get(ddmFormField.getName())), LanguageUtil.get(ddmFormViewFormInstanceRecordsDisplayContext.getDefaultLocale(formInstanceRecord), "true")) %>'>
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/form_instance_records_search_container.jsp:128]: @mlan0, why did you decide to invoke the method `getDefaultLocale`? Will it work as intended if the user's language is Spanish, for example? Do you remember our discussion about portal languages (instance language, site language and user language)?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMImpl.java [hunk 2]
```diff
@@ -283,10 +283,11 @@ else if (type.equals(DDMFormFieldType.CHECKBOX)) {
 			Boolean valueBoolean = (Boolean)fieldValue;
 
 			if (valueBoolean) {
-				fieldValue = LanguageUtil.get(themeDisplay.getLocale(), "yes");
+				fieldValue = LanguageUtil.get(themeDisplay.getLocale(), "true");
 			}
 			else {
-				fieldValue = LanguageUtil.get(themeDisplay.getLocale(), "no");
+				fieldValue = LanguageUtil.get(
+					themeDisplay.getLocale(), "false");
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMImpl.java:290]: @mlan0, did you check if this change affected System Settings?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/service/dependencies/readonly/checkbox.ftl [hunk 3]
```diff
@@ -17,9 +17,9 @@
 		</#if>
 
 		<#if stringUtil.equals(fieldValue, "true")>
-			<@liferay_ui.message key=escape("yes") />
+			<@liferay_ui.message key=escape("true") />
 		<#else>
-			<@liferay_ui.message key=escape("no") />
+			<@liferay_ui.message key=escape("false") />
```
Comment (natocesarrego) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/com/liferay/dynamic/data/mapping/service/dependencies/readonly/checkbox.ftl:22]: @mlan0, did you check if this change affected System Settings?

**Comment by natocesarrego** [comment]
Yesterday, Mateus and I talked about my first comment, although I'm still waiting an answer for my last 2 comments. I've already reached out him through Slack this morning.

**Comment by natocesarrego** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:reevaluate:1167401_8650

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:reevaluate:1307401_340

**Comment by carolmariaabb** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**